udp-ballerina/connectionless_client.bal

```diff
@@ -24,18 +24,18 @@ public client class Client {
     private string? localHost = ();
    
     # Initializes the UDP connectionless client based on the provided configurations.
+    # ```ballerina
+    # udp:Client|udp:Error? socketClient = new(localHost = "localhost");
+    # ```
     #
-    # + localHost - Local binding of the interface
-    # + timeoutInMillis - The socket reading timeout value to be used in milliseconds. If this is not set,
-    #                         the default value of 300000 milliseconds (5 minutes) will be used.
-    public isolated function init(string? localHost = (), int timeoutInMillis = 30000) returns Error?  {
-        self.localHost = localHost;
-        return initConnectionlessClient(self, localHost, timeoutInMillis);
+    # + config - Connectionless client related configuration
+    public isolated function init(*ClientConfig config) returns Error? {
+        return initConnectionlessClient(self, config);
     }
 
     # Sends the given data to the specified remote host.
     # ```ballerina
-    # udp:Error? result = socketClient->send({remoteAddress: {host: "localhost", port: 48826}, data:"msg".toBytes());
+    # udp:Error? result = socketClient->send({remoteHost: "localhost", remotePort: 48826, data:"msg".toBytes()});
```

Comment:
```suggestion
    # udp:Error? result = socketClient->sendDatagram({remoteHost: "localhost", remotePort: 48826, data:"msg".toBytes()});
```
Have to change the method name as updated in the design doc.

Reply:
updated